packages/esm-appointments-app/src/form/appointments-form.scss

```diff
@@ -97,17 +76,3 @@ $openmrs-background-grey: #ededed;
   max-inline-size: fit-content;
   padding-top: spacing.$spacing-03;
 }
-
-:global(.cds--select-input):focus {
-  outline: 2.5px solid colors.$red-60;
-}
```

Comment:
This (and the one similar ones below) were the problematic cases.  They existed previous to the refactor, but one result of the refactor was that the appointment form was being used in multiple places, so there are now more times when this global override was being applied.  

It looked like the main use case here was to make sure a field was highlighted in red when the form submittal handler set focus on the input (say if for instance it was a mandatory field that had not been selected).  I considered adding a wrapper or something so that this would *only* be applied to inputs on this form, but I realized that we probably don't want to apply this non-standard UI pattern.  Now, when a mandatory field does not have a value, it still switched focus to that field and the highlights it, just in the default outline color, not red.